src/components/appSwitch/AppSwitch.js

```diff
@@ -19,7 +19,7 @@ import config from '../../config/config'
 import { delay } from '../../lib/utils/async'
 import DeepLinking from '../../lib/utils/deepLinking'
 import { isMobileNative } from '../../lib/utils/platform'
-import restart from '../../lib/utils/restart'
+import { restart } from '../../lib/utils/system'
```

Comment:
why those changes are  displayed if you've rebased over the latest master ?

Reply:
This change is here because while pushing this file was giving an error and I resolved it